### ISSUE TITLE
Add a way to search stdout/stderr

### ISSUE THREAD

**Issue by theres-waldo** [Pernosco/pernosco#4]
I often start a debugging session by finding a relevant / interesting line in the output. Having to press the "more" button in the stdout/stderr view a hundred times to see all the output inhibits this.

**Comment by khuey** [comment]
We've been contemplating adding a "find" feature. I presume that would get the job done?

**Comment by theres-waldo** [comment]
What would it do if the result is in middle of the collapsed section -- expand it all? Or create two collapsed sections to either side of the result?

My personal preference would be to have a way to show all the output, which can then be searched using the browser's search. If the browser search is too slow, then the site could provide a faster search as a bonus.

**Comment by rocallahan** [comment]
The problem is that "all the output" could be a really huge amount of output that, when added to the DOM, makes performance terrible.

**Comment by theres-waldo** [comment]
This is true. Let me morph this into two issues, then: (1) searching stdout/stderr, and (2) downloading stdout/stderr for processing elsewhere.

**Comment by theres-waldo** [comment]
Filed #5 for downloading.
